terracotta/scripts/ingest.py

```diff
@@ -68,6 +71,9 @@ def push_to_last(seq: Sequence[Any], index: int) -> Tuple[Any, ...]:
 
     if not output_file.is_file():
         driver.create(keys)
+    elif skip_existing:
```

Comment:
This could just be `if skip_existing`.

Reply:
Why would you prefer that? For clarity? I guess there is no point in checking for existing datasets when the db has just been created?

Reply:
It's just something I stumbled over when I read your changes. I had to stop and think why this would only apply in the `else` case. IMO it also improves maintainability; you could change or remove the first check without impacting the second.

Reply:
It's a nitpick, of course :)